phc-rails/db/schema.rb

```diff
@@ -37,20 +36,23 @@
     t.string   "How_long_have_you_been_homeless__c",        limit: 255
     t.string   "Where_do_you_usually_go_for_healthcare__c", limit: 255
     t.string   "Medical_Care_Other__c",                     limit: 255
+    t.date     "Birthdate__c"
   end
 
   add_index "accounts", ["FirstName"], name: "index_accounts_on_FirstName", using: :btree
   add_index "accounts", ["LastName"], name: "index_accounts_on_LastName", using: :btree
-  add_index "accounts", ["sf_id"], name: "index_accounts_on_sf_id", unique: true, using: :btree
 
   create_table "event_registrations", force: :cascade do |t|
-    t.string   "account_sfid", limit: 255
-    t.string   "phc_sfid",     limit: 255
-    t.string   "FirstName",    limit: 255
-    t.string   "LastName",     limit: 255
+    t.string   "account_id",         limit: 255
+    t.string   "phc_sfid",           limit: 255
+    t.string   "FirstName",          limit: 255
+    t.string   "LastName",           limit: 255
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.